docs/index.rst

```diff
@@ -2,19 +2,47 @@
 ops library API reference
 =========================
 
+The `ops` library is a Python framework for writing and testing Juju charms.
+
+For more about the Charm SDK, see https://juju.is/docs/sdk.
+
+The library provides:
+
+- :ref:`ops_main_entry_point` used to initialise and run your charm.
+- :ref:`ops_module`, the API to respond to Juju events and manage the application.
+- :ref:`ops_pebble_module` low-level API for Pebble in Kubernetes containers.
+- :ref:`ops_testing_module`, the framework for unit testing charms in a simulated environment.
+
+You can write a charm in any way you want, but with the `ops` library you get a
+framework that helps you write consistent readable charms following the latest
+best practices, reuse code via charm libs, and separate typical charm concerns
+--- application state management from integration management or lifecycle
+management from testability.
+
+Whether you are a machine or a Kubernetes charm author, `ops` is the recommended way to go.
+
```

Comment:
I know this was my wording, but here's perhaps a better rephrase:

> Whether your charm is for machines or Kubernetes, ...

Reply:
done.